packages/presentation/lib/screen/home/widgets/home_tab_bar.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:presentation/config/theme/app_colors.dart';
+
+class HomeTabBar extends StatefulWidget {
+  const HomeTabBar({Key? key}) : super(key: key);
+
+  @override
+  State<HomeTabBar> createState() => _HomeTabBarState();
+}
+
+class _HomeTabBarState extends State<HomeTabBar> {
+  int currentIndex = 0;
+  bool isIconActive = false;
+
+  @override
+  Widget build(BuildContext context) {
+    return TabBar(
+      splashBorderRadius: BorderRadius.circular(16),
+      onTap: (index) {
+        setState(() {
+          if (index == currentIndex) return;
+          currentIndex = index;
+          isIconActive = !isIconActive;
+        });
+      },
```

Comment:
what are you using "onTap" property for if you are already using "DefaultTabController"? If to move the icon, then when you swipe it still remains in place. Fix movement when using swipe or remove it altogether

Reply:
removed this logic after discussion with Sergey